CHANGELOG.yaml

```diff
@@ -1,13 +1,16 @@
 master:
+  chores:
+    - >-
+      GH-1099 Added unit tests for mutation-tracker.js, property-list.js, request.js, url.js
+      and variable.js.
+    - GH-1098 Added unit tests for cookie.js, header.js, request-auth.js, item.js, content-info.js.
```

Comment:
Let's follow this order in the changelog:
- new features
- breaking changes
- fixed bugs
- chores

Reply:
Also, combine these two and let just have:
`GH-1098, GH-1099 Added unit tests`

Reply:
Done